tests/common/include/tests/common/tools_propagator.inl

```diff
@@ -150,18 +249,21 @@ TEST_P(PropagatorWithRkStepper, propagator_rk_stepper) {
             free_track_parameters traj(ori, 0, mom, -1);
             traj.set_overstep_tolerance(-10 * unit_constants::um);
 
-            helix_gun helix(traj, B);
-            combined_inspector ci{helix_inspector(helix),
-                                  propagation::print_inspector{}};
+            std::get<helix_inspector::state_type &>(actor_states)
+                ._helix.init(traj, &B);
```

Comment:
I think you have to use the original constructor for `helix_gun`. Or is there any technical issue with that

Reply:
See comment :) I just could not get it to work back then. Let's have another try

Reply:
Ok, it works now. I guess, using a reference to the b field might have also deleted the copy assignment operator previously. Thanks for pointing out!